chemicaltoolbox/fragmenter/merger.xml

```diff
@@ -31,19 +30,7 @@
         <data format="smi" name="outfile" label="Fragment merging on ${on_string}"/>
     </outputs>
     <tests>
-        <test>
-            <param name="infile" ftype="sdf" value="CID_2244.sdf"/>
-            <param name='rules' value='recap' />
-            <param name='oformat' value='smi' />
-            <output name="outfile" ftype="smi" file='fragmenter_on_CID_2244_recap.smi'/>
-        </test>
-        <test>
-            <param name="infile" ftype="sdf" value="CID_2244.sdf"/>
-            <param name='rules' value='recap' />
-            <param name='oformat' value='smi' />
-            <param name='mark' value='--mark-sticky-ends' />
-            <output name="outfile" ftype="smi" file='fragmenter_on_CID_2244_recap_mark.smi'/>
-        </test>
+        <test expect_failure="true" />
```

Comment:
Its not worth the effort, but we should also not push tools that fail, lets remove the .shed.yml and not publish this tool

Reply:
Well the planemo lint complains if there is no shed file if I am not mistaken.

Reply:
Yes, do that as very last step. I need to push then manually the good tools.